src/docs/implementation/SwateManual/Docs04-Ontology-Term-Search.md

```diff
@@ -0,0 +1,89 @@
+- You can also use the advanced term search for a more refined search approach. Just click on the blue link close to the related search field.
+
+<br>
+
+# Term Insert
+
+<br>
+
+<p style="display: flex; justify-content: center">
+<img src="./../../img/Swate-TermInsert-Exp.jpg?v01.02.22" style="height: 300px">
+</p>
+
+
+You can find this function on the *Terms* tab in the <a href="./../../img/Swate-Overlay-Exp.jpg" target="_blank">navbar</a>. Use this function to insert terms in an [ISA-Tab conform way](https://isa-specs.readthedocs.io/en/latest/isatab.html#ontology-annotations), for example for Swate building blocks.
+
+- Type in the search field to trigger the auto-complete function and find terms. Select the term you want or follow the information under [General](#General) for more information.
+- You can now insert the chosen term anywhere in Excel, not only inside of a Swate [annotation table](./Docs02-Annotation-Table.html).
+    - Terms will be added to all selected rows in the format:
+
+        `| term name | term source ontology | purl.obolibrary.org link for term |`
+        
+        This always done for the next <u>three</u> Excel columns including the selected column.
+
+<p style="display: flex; justify-content: center">
+<img src="./../../img/Swate-TermInsert2-Exp.jpg?v01.02.22" style="height: 100px">
+</p>
+
+- You can also click on the *Copy to Clipboard*-button next to the insert term-button. This will copy the term information (name, source ontology, purl link) to your clipboard and you can paste it into Excel with <kbd>Ctrl</kbd> + <kbd>V</kbd>. This will enable you to paste the information spread over <u>three</u> rows instead of columns. This works only for a single column.
+
+<br>
+
+# Relationship-Directed Search
+
+## Relationship-Directed Search Theory
+
+Ontology terms contain different types of relationships between each other. For example `is_a`, `part_of`, `xref` relationships. Swate and the SwateDB utilize these relationships to provide the user with a predefined set of terms as most likely search requests. 
+
+Let's look at this with an example:
+
+| Source Name | Component [instrument model] | Term Source REF (MS:1000031) | Term Accession Number (MS:1000031) | Sample Name |
+|-------------|------------------------------|------------------------------|------------------------------------|-------------|
+|             |                              |                              |                                    |             |
+
+If a user wants to fill in `Component [instrument model]` then they are most likely looking for the instrument model used in their experimental workflow. Here for example the instrument model series and the exact instrument models are connected to `instrument model (MS:1000031)` via [`is_a` relationships](/images/UserDocs/Swate-RelationshipGraph-Exp.jpg). 
```

Comment:
`/images/UserDocs/Swate-RelationshipGraph-Exp.jpg` link is broken

Reply:
fixed